lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -143,13 +142,17 @@
     required: false
     default: 0
     description:
-    - Service memory limit in MB. 0 equals no limit.
+    - Service memory limit. 0 equals no limit. Accepts a float value
+      representing bytes or a string value with a units identification
+      char (100000b, 1000k, 128m, 1g).
```

Comment:
Please copy the texts from `docker_container`'s `kernel_memory` option. We've tried to unify these texts for `docker_container` already, it would be nice if other `docker_*` modules would follow suit. :)

Reply:
Perfect! I've updated the docs. Looks like `docker_container` is doing conversion to support `KiB`, `MiB` and so on. Since this module does not support this I changed it to describe `KB` and `MB` instead. The docker API supports these values out of the box.

Reply:
Both modules use the same code ([human_to_bytes](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py#L580-L625) from module_utils/basic.py); see https://github.com/ansible/ansible/blob/dae35ccd34808849d0aa8acb0e46ac040354f50f/lib/ansible/modules/cloud/docker/docker_swarm_service.py#L617

We cannot use the Docker API's conversion since that would break idempotence.

Reply:
Ah, of course. I'll use the verbatrim kernel_memory description from docker_container.